packages/cli/src/generators/app.ts

```diff
@@ -81,29 +85,27 @@ class AppGenerator extends Generator<AppGeneratorOptions> {
       throw new Error('Failed running prettier')
     }
 
-    // TODO: someone please clean up this ugly code :D
     // Currently aren't failing the generation process if git repo creation fails
-    const gitResult1 = spawn.sync('git', ['init'], {
-      stdio: 'ignore',
-    })
-    if (gitResult1.status === 0) {
-      const gitResult2 = spawn.sync('git', ['add', '.'], {
-        stdio: 'ignore',
-      })
-      if (gitResult2.status === 0) {
-        const gitResult3 = spawn.sync('git', ['commit', '-m', 'New baby Blitz app!'], {
-          stdio: 'ignore',
-        })
-        if (gitResult3.status !== 0) {
-          console.error('Failed to run git commit')
-        }
-      } else {
-        console.error('Failed to run git add')
-      }
+    if (gitInitResult.status === 0) {
```

Comment:
Curious on the motivation behind not failing if the `git` commands fail. If we want to i am happy to make them throw error here as well as the other commands inside this generator.

Reply:
I'm not super set on failing if git init fails just because it's not a mandatory part of the setup process. I'll open an issue to discuss this more though, thanks for bringing it up!